mmif/utils/mmif_rewinder.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import mmif
+import json
+import os
+
+def read_mmif(mmif_file)->mmif.Mmif:
+    """
+    Function to read mmif file and return the mmif object.
+    (Would it be better to be a mmif object?)
+
+    :param mmif_file: file path to the mmif.
+    :return: mmif object
+    """
+    try:
+        with open(mmif_file, 'r') as file:
+            mmif_data = json.load(file)
```

Comment:
`mmif.Mmif.__init__` can directly de-serialize `str` representation of the MMIF JSON content, so using `json.load` is unnecessary.